### ISSUE TITLE
AsyncPackage for VS2015 and later

### ISSUE THREAD

**Issue by duncansmart** [bennor/AutoT4MVC#20]
VS 2019 RC is showing this warning now:
![image](https://user-images.githubusercontent.com/576343/53945325-46a72f80-40b9-11e9-98ec-0742ebdb3001.png)

The guidance is to move to the new `AsyncPackage` which was introduced in VS2015. This will make AutoT4MVC incompatible with VS2013 and earlier though. 

I will happily create a PR for `AsyncPackage` - or do we want to maintain support for VS2013? Or do we want two versions to support both?

**Comment by bennor** [comment]
I don't think we really need to support below VS 2015 anymore. I would happily accept this PR.

**Comment by bennor** [comment]
Actually, thinking about it now, it could be a problem. I don't want people to not be able to install the old version, and I think that might happen if I push out a new version without support.

Any way round it?

Worst case I guess I could fork and add a new version for VS 2019.
